### ISSUE TITLE
Support model instance save?

### ISSUE THREAD

**Issue by proofit404** [M1hacka/django-pg-returning#5]
Hi! Thanks for an awesome Django extension!

Let's consider we have code like this:

```python
community = Community.objects.get(pk=1)
community.post_number = F("post_number") + 1
community.save(update_fields=["post_number"])

community.refresh_from_db() # we need this to fetch result of the F + 1 expression.
```
I propose to include a ModelMixin with `returning_save` method.
It will reset model instance attributes from the database result. It should use fields mentioned in the `update_fields` or all of them.

The code above will be rewritten like this without extra DB query:

```python
community = Community.objects.get(pk=1)
community.post_number = F("post_number") + 1
community.returning_save(update_fields=["post_number"])
```

Best regards,
Artem.

**Comment by M1ha-Shvn** [comment]
Interesting. I haven't thought of such a case.
Though, `save()` method is much more complex, as I remember, and can't be implemented in the way I do it with `update()` and `delete()`. I'll have a look on weekend or you can make a PR if you want it faster.
P. s. it's better to name method `save_returning` as other methods in this library have postfix names.
P. p. s. the case above may be implemented using simple `update_returning`:
```python
qs = Community.objects.filter(pk=1).only('post_number').update_returning(post_number=F('post_number') + 1)
community.post_number = qs[0].post_number
```

**Comment by proofit404** [comment]
Thanks for the code snippet. I'll use it for now :tada:

**Comment by M1ha-Shvn** [comment]
Released 1.1.0.
Note, that integration process have changed a little bit. Now you need to inherit UpdateReturningModel in order to have `save_returning()` method for your instances.

**Comment by proofit404** [comment]
Thank you so much!
